remotecv/detectors/feature_detector/__init__.py

```diff
@@ -7,17 +7,20 @@
 # Licensed under the MIT license:
 # http://www.opensource.org/licenses/mit-license
 # Copyright (c) 2011 globo.com timehome@corp.globo.com
+import datetime
 
 import cv2
 import numpy as np
 
 from remotecv.detectors import BaseDetector
+from remotecv.utils import context
 
 # pylint: disable=no-member
 
 
 class FeatureDetector(BaseDetector):
     def detect(self, image):
+        start_time = datetime.datetime.now()
```

Comment:
Do you think we can change this to use [time.perf_counter_ns](https://docs.python.org/3/library/time.html#time.perf_counter_ns)? It's a better way of tracking start and end time as it is monotonic.

Then to get seconds, after subtracting start from end, we can just divide by 1_000_000_000.

Reply:
Thanks for the ideias. But IMHO we should keep the same way thumbor get this time métrics, like [here](https://github.com/thumbor/thumbor/blob/3601a01f63ab4eba159abbd8770d236163b0fe16/thumbor/handlers/__init__.py#L81). What do you think?

Reply:
I think thumbor is doing it incorrectly, simply because at the time we didn't have this API (thumbor is 10+ years old). I'd rather we fix it bit by bit to be more precise than just keep going as it is. If you feel strongly that this will take much of your time, we can move ahead to do this in a separate PR, but I felt like it's a small change.

Reply:
Thanks for tackling this! I really appreciate it, but just to make sure we won't have to do it again in the future, can you create a module `timings.py` with two functions `get_time()` and `get_interval(t1, t2)`? This way if in the future we need to change how we're tracking time (again) we don't have to go scouring the entire codebase for it.

Reply:
Nicee, i totatly agree. What about now?

Update: pipeline is broken, im working on it! (accept help 🙃)

Reply:
Nicee, i totally agree. What about now @heynemann?